apps/payments/next/app/[locale]/[offeringId]/[interval]/landing/route.ts

```diff
@@ -137,14 +138,23 @@ export async function GET(
   if (searchParams?.spVersion) {
     delete searchParams.spVersion;
   }
+  const session = await auth();
 
-  const { taxAddress } = await getTaxAddressAction(ipAddress);
+  const fxaUid = session?.user?.id;
```

Comment:
On this `/landing` route the no prompt auth hasn't occurred yet, so the session should always be undefined. (Unless maybe there was a previous session, but that wouldn't be the typical user flow)

Reply:
Should we move this redirect after the auth prompt in that case?